typescript/package.json

```diff
@@ -25,6 +25,8 @@
   "dependencies": {
     "axios": "^0.21.1",
     "http-status-codes": "^2.1.4",
+    "kurtosis-core-api-lib": "^0.13.0",
```

Comment:
Rather than directly depending on kurtosis-core-client, this should pull in `kurtosis-testsuite-api-lib` (the direct dependency), which will transitively pull in `kurtosis-core-client`